.bazelrc

```diff
@@ -1,5 +1,5 @@
 # build config
-build --compilation_mode=opt
+build --compilation_mode=dbg
```

Comment:
Accident?

Reply:
Yes, this was for debugging. Will revert back.